part-1-ajax/test.js

```diff
@@ -1,76 +1,174 @@
-function getCountryName(code) {
-    $.ajax({
-      //uses the three letter code to get the country's full name
-      }
-    })
-  }
-
-  function parseData(country) {
-    //takes the country data
-    //extracts the name, capital, population, and neighboring countries
-    //iterates through the neighbors array to send off second AJAX request
-  }
-
-  var button = document.getElementsByTagName('button')[0]
-  button.addEventListener('click', function(){
-
-    $.ajax({
-
-    })
-  })
-
-//When you search for a country you should receive a response that looks like:
-[ {
-      "name": "Argentina",
-      "topLevelDomain": [
-          ".ar"
-      ],
-      "alpha2Code": "AR",
-      "alpha3Code": "ARG",
-      "currencies": [
-          "ARS"
-      ],
-      "callingCodes": [
-          "54"
-      ],
-      "capital": "Buenos Aires",
-      "altSpellings": [
-          "AR",
-          "Argentine Republic",
-          "República Argentina"
-      ],
-      "relevance": "0",
-      "region": "Americas",
-      "subregion": "South America",
-      "languages": [
-          "es",
-          "gn"
-      ],
-      "translations": {
-          "de": "Argentinien",
-          "es": "Argentina",
-          "fr": "Argentine",
-          "ja": "アルゼンチン",
-          "it": "Argentina"
-      },
-      "population": 43131966,
-      "latlng": [
-          -34,
-          -64
-      ],
-      "demonym": "Argentinean",
-      "area": 2780400,
-      "gini": 44.5,
-      "timezones": [
-          "UTC−03:00"
-      ],
-      "borders": [
-          "BOL",
-          "BRA",
-          "CHL",
-          "PRY",
-          "URY"
-      ],
-      "nativeName": "Argentina"
-  }
-]
+$('.button').on('click', getCountry);
+
+// CREATE THE FUNCTION
+function getCountry(data) {
+
+// PREVENT THE DEFAULT BEHAVIOR 
+data.preventDefault();
+
+// GET THE INPUT FROM THE SEARCH BOX AND LOG IT TO THE CONSOLE
+var ourInput = $('.input').val();
+
+var country = ourInput.split(' ').join('%20')
+
+	// OKAY, NOW WE MAKE OUR FIRST AJAX CALL TO GET THE COUNTRIES
+	$.ajax({
+		url: "https://restcountries.eu/rest/v1/name/" + country + "?fullText=true",
+		success: returnCountryCode
+	})
+
+	// APPEND THE COUNTRY NAME
+	var countryName = data[0].name.toString();
+	$('p').remove();
+		for(i=0; i < data.length; i++) {
+			var conName = document.createElement('p')
+			conName.innerHTML = people
+			$('.name').append(conName)
+		}
+
+
+	// CREATE THE GET CAPITAL NAME FUNCTION
+	function getCapital(capital) {
```

Comment:
i think you're writing too many functions here, don't think you need these functions to add on the capital, population and name.